package.json

```diff
@@ -40,16 +40,16 @@
     "**/qunit": "^2.14.1"
   },
   "devDependencies": {
-    "@types/jest": "^24.0.11",
+    "@types/jest": "^26.0.23",
     "@typescript-eslint/eslint-plugin": "^4.1.1",
     "@typescript-eslint/parser": "^4.1.1",
     "cross-env": "^7.0.3",
     "eslint": "^7.14.0",
-    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-prettier": "^8.3.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-prettier": "^3.1.4",
-    "jest": "^24.5.0",
-    "prettier": "2.1.2",
+    "jest": "^24.9.0",
+    "prettier": "2.3.1",
```

Comment:
Hmm, why do we pin? I'd expect `^` here and then leveraging the `yarn.lock` for the pinning.

Reply:
ya, pinning seems odd. I can make it drift